grammars/go.cson

```diff
@@ -85,7 +85,7 @@
   }
   {
     'name': 'storage.type.go'
-    'match': '\\b(chan|map|bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
+    'match': '\\b(bool|string|error|int|int8|int16|int32|int64|rune|byte|uint|uint8|uint16|uint32|uint64|uintptr|float32|float64|complex64|complex128)\\b'
```

Comment:
If you look it this with the theme I'm using, the current scoping still makes a lot more sense to me:

![image](https://cloud.githubusercontent.com/assets/4171547/8872146/cdb04d44-31fa-11e5-9832-c2c0c5f11f0f.png)
It makes very clear that I'm using the types `map` and `chan`. If you make them as keywords they fade away together with all the other keywords making the same line harder and less clear to read:

![image](https://cloud.githubusercontent.com/assets/4171547/8872187/17b7abd0-31fb-11e5-8a48-c91edb8b400e.png)

So IMO this should really be scoped as `storage.type.go`.

@joefitzgerald what's your take on this specific one? It seems the language spec leaves room for both `keyword.go` and `storage.type.go` to be used. So it comes down to usability and personal preference I think...